custom-docker/README.md

```diff
@@ -0,0 +1,156 @@
+
+Use it as a base to create your own custom image suitable for your needs with
+different deployment use cases and clusters. 
+
+The following sections contain information for building, using, and customizing
+the image.
+
+## General information
+
+Users can use the Trino Docker image directly, with Docker compose, with the
+[Trino Helm charts](https://trinodb.github.io/charts/) or other deplopyment
+tools.
+
+The Trino project provides a `trinodb/trino-core` image tarball with the minimal
+plugins and a default `trinodb/trino-core` image with all available plugins.
+Both images also include the Trino CLI and can be started successfully without
+further configuration. Details for running and using the image are available in
+the
+[Trino documentation](https://trino.io/docs/current/installation/containers.html).
+
+This project uses the core image and adds necessary configuration files and a
+limited subset of plugins to create a image with the following characteristics:
+
+* Catalog `abyss` using the Blackhole connector.
+* Catalog `brain` using the Memory connectors, configured for UDF storage in the
+  `default` schema, and `brain.default` is added to the SQL path.
+* Catalog `generator` using the Faker connector.
+* Catalog `llm` using the AI functions plugin, the `llm.ai` schema location is
+  added to the SQL path so AI functions can be invoked by name alone. Requires a
+  locally running Ollama.
+* Catalog `monitor` using the JMX connector.
+* Catalog `tpch` using the TPC-H connector.
+* Catalog `tpcds` using the TPC-DS connector.
+* Node configured to act as coordinator and worker to allow single node use.
+* Environment name set to `custom` in `node.properties`.
+* No additional logging configured in `log.properties`.
+* Preview Web UI enabled.
+* `geospatial` functions plugin removed.
+* `exchange-filesystem` plugin removed.
+
+The project is configured for Trino 472.
+
+## Building
+
+Build requirements:
+
+* Linux or MacOS with bash shell (others might work but are untested)
+* Docker (or equivalent tool)
+
+Download and extract or clone the repository to work with the `custom-docker`
+directory locally on your machine.
+
+Run the build script:
+
+```shell
+cd custom-docker
+./build.sh
+```
+
+The project uses the `trino-core` Docker images for the the configured Trino
+release, adds configurations files from `customization` and a limited number of
+plugins configured in `processPluginsForCustomizations` in `build.sh`, and
+repackages it into a Docker image.
+
+After a successful build, you find the Docker images for the three different
+processor architectures locally with `docker images`:
+
+```
+REPOSITORY     TAG           IMAGE ID       CREATED              SIZE
+trino-custom   474-ppc64le   2279b0a14cb6   About a minute ago   790MB
+trino-custom   474-arm64     67a3dcee3c85   About a minute ago   765MB
+trino-custom   474-amd64     df994bb44819   About a minute ago   757MB
```

Comment:
Should we also create a multiarch manifest containing all these images? Like here: https://github.com/trinodb/release-scripts/blob/master/trino-docker.sh#L22

Reply:
I added a link to that script in the section where I talk about tag and push..